src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -15,17 +15,19 @@ public OperateToCommand(int posX, int posY) {
 		this.posY = posY;
 	}
 
+	public void movePoint(int x, int y) {
+		this.posX += x;
+		this.posY += y;
+	}
+
```

Comment:
create your own implementation of MutableOperateToCommand. You can do it in src/test/java.